apps/web/modules/ui/components/sidebar/index.test.tsx

```
@@ -0,0 +1,584 @@
+}));
+
+// Test component that uses useSidebar hook
+const TestComponent = () => {
+  const sidebar = useSidebar();
+  return (
+    <div>
+      <div data-testid="sidebar-state">{sidebar?.state || "unknown"}</div>
+      <div data-testid="sidebar-open">{sidebar?.open?.toString() || "unknown"}</div>
+      <div data-testid="sidebar-mobile">{sidebar?.isMobile?.toString() || "unknown"}</div>
+      <div data-testid="sidebar-open-mobile">{sidebar?.openMobile?.toString() || "unknown"}</div>
+      <button data-testid="toggle-button" onClick={sidebar?.toggleSidebar}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)